client/sites-dashboard-v2/site-preview-pane/dotcom-preview-pane.tsx

```diff
@@ -45,10 +46,20 @@ const DotcomPreviewPane = ( {
 				setSelectedSiteFeature,
 				selectedSiteFeaturePreview
 			),
+			createFeaturePreview(
+				DOTCOM_DEVELOPER_TOOLS_PROMO,
+				<span>
+					{ __( 'Dev Tools' ) } <DevToolsIcon />
+				</span>,
+				! isAtomicSite,
```

Comment:
Do we want to display it on a self-hosted site?

Reply:
Ahh good question, at first I thought we do want it but reading the issue again, it says 

> Jetpack: keep the same behavior as now.

which I think would mean having the two existing panels? right @allilevine